README.md

```diff
@@ -9,14 +9,32 @@ please see the
 repository.
 
 
-## Set up the ODBC driver
-1. If you are not on a Windows machine, install the [unixODBC](http://www.unixodbc.org/) driver manager. Windows users can utilize the built-in driver manager.
-2. Update the values of `Driver`, `Pwd`, `Server`, `User`, and `Database` in `setup/setupDSN.reg`. The value of `Driver` should be the absolute path of `mongo-odbc-driver/target/debug/mongoodbc.dll`.
-3. Open the registry editor and add a new key under `ODBC/ODBCINST.INI` called `ADL_ODBC`. Then, add two sub-entries to `ADL_ODBC`:
+## Set up the ODBC driver on Windows (architecture-agnostic)
+Note: users can utilize the built-in driver manager.
+1. Update the values of `Driver`, `Pwd`, `Server`, `User`, and `Database` in `setup/setupDSN.reg`. The value of `Driver` should be the absolute path of `mongo-odbc-driver/target/debug/mongoodbc.dll`.
```

Comment:
The lib can be built as debug or release.
Maybe just say :
The absolute path of the `mongoodbc.dll` library. If you just built it, check in your `mongo-odbc-driver/target/` debug or release directories.